docs/source/developer_guide/git_workflow.rst

```diff
@@ -57,7 +57,7 @@ Creating a fork - Initial one time setup
        > upstream  https://github.com/sktime/sktime.git (push)
 
 
-Contributing a feature - for every new feature
+Developing a feature
```

Comment:
same, "for every new feature" is needed to signpost which section is repeatable, which is one-time